Source/Scene/BillboardCollection.js

```diff
@@ -255,6 +266,16 @@ define([
                 return that._textureAtlas.texture;
             }
         };
+
+        this._removeEventFunc = undefined;
+        if (defined(this._globe)) {
+            this._removeEventFunc = this._globe._surface.tileRenderedEvent.addEventListener(function(tile) {
```

Comment:
Do you think billboard's should use the tile visible event directly?  I don't think this is the level of abstraction billboards should work with.  A billboard just wants to subscribe to the height for a long/lat.  We could keep this intermediate abstraction private for now.